components/02-elements/form/select/_select.scss

```diff
@@ -21,6 +21,10 @@ $select__field-list-border-radius--single: 20px !default;
             border: $select__border;
             border-radius: $select__border-radius;
             appearance: none;
+
```

Comment:
I think we can add `transition: $transition-base` here in `select`, `input` and `textarea` fields. I think it looks nicer with small animation :)